src/Inputs/Timepicker/index.tsx

```diff
@@ -13,7 +13,7 @@ export interface TimepickerProps extends LabelLayoutProps {
     onChange?: Function;
     disabled?: boolean;
 }
-// @ts-ignore not sure why this is being highlighted as an issue
+// @ts-ignore TimePicker throwing error because of withTheme
```

Comment:
https://github.com/mui-org/material-ui/issues/8070 does this solve it?

Reply:
@fpunny Could you explain what withTheme is being used for in Timepicker?